### PR TITLE
refactor(value-list): tailwindify

### PR DESCRIPTION
**Pull request by julio8a**
Supports #1500

### CODE REVIEW

File: src/components/calcite-value-list/calcite-value-list.scss [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-  @extend %component-spacing;
-  align-items: stretch;
-  background-color: transparent;
-  display: flex;
+  @apply items-stretch 
+    bg-transparent 
+    flex 
+    flex-col 
+    relative;
   flex: 0 0 auto;
```
Comment (jrowlingson) [inline src/components/calcite-value-list/calcite-value-list.scss:8]: `flex-shrink-0 flex-grow-0` ?

File: src/components/calcite-value-list/calcite-value-list.scss [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
 
 calcite-filter {
-  margin-bottom: 1px;
+  @apply mb-px;
 }
 
 calcite-scrim {
-  display: flex;
+  @apply flex flex-col pointer-events-auto;
   flex: 0 0 auto;
```
Comment (jrowlingson) [inline src/components/calcite-value-list/calcite-value-list.scss:33]: See above.

File: src/components/calcite-value-list/calcite-value-list.scss [hunk 3]
```diff
@@ -1,39 +1,34 @@
...
 }
 
 calcite-filter {
-  margin-bottom: 1px;
+  @apply mb-px;
 }
 
 calcite-scrim {
-  display: flex;
+  @apply flex flex-col pointer-events-auto;
```
Comment (jrowlingson) [inline src/components/calcite-value-list/calcite-value-list.scss:32]: `pointer-events-none` or was this intended?

Comment (julio8a) [reply src/components/calcite-value-list/calcite-value-list.scss]: good catch! fixed.

**Comment by julio8a** [comment]
anything else here?

**Review by jrowlingson — APPROVED** [review]

**Pull request merged**